uaclient/util.py

```diff
@@ -211,7 +211,8 @@ def readurl(url: str, data: 'Optional[bytes]' = None,
 def _subp(args: 'Sequence[str]',
           rcs: 'Optional[List[int]]' = None,
           capture: bool = False,
-          timeout: 'Optional[float]' = None) -> 'Tuple[str, str]':
+          timeout: 'Optional[float]' = None,
+          decode: 'Optional[bool]' = True) -> 'Tuple[str, str]':
```

Comment:
This type signature is inaccurate, as we can now return str or bytes as the tuple members. Similar to below, I don't love the return type of a function being dynamic based on one of its arguments, because it makes the codebase harder to reason about, but it's more of a PITA to address here than there because of the multiple layers we have going on.

Reply:
Dropped these changes, from util, turns out I don't need them while using gpg --output thanks for that.